src/components/AddItem.js

```diff
@@ -68,53 +68,106 @@ const AddItemForm = props => {
                 <option value="14">Kind of Soon</option>
                 <option value="30">Not soon</option>
-              </select>
-              <div className="pointer-events-none absolute inset-y-0 right-0 flex items-center px-2 text-gray-700">
-                <svg
-                  className="fill-current h-4 w-4"
-                  xmlns="http://www.w3.org/2000/svg"
-                  viewBox="0 0 20 20"
-                >
-                  <path d="M9.293 12.95l.707.707L15.657 8l-1.414-1.414L10 10.828 5.757 6.586 4.343 8z" />
-                </svg>
-              </div>
+              </select> */}
+            <div className="flex flex-col items-center sm:flex-row">
+              <label
+                className="bg-transparent text-center mt-2 border border-b-4 border-green-600 hover:bg-green-600 hover:text-white active:border-green-400 rounded-lg py-2 px-4 w-40 sm:w-full sm:mx-2"
+                checked
+                value={enteredValue.nextPurchase}
+              >
+                Soon
+                <input
+                  className="radio"
+                  type="radio"
+                  name="nextPurchase"
+                  value="7"
+                  onChange={handleChange}
+                />
+              </label>
+              <label
+                className="bg-transparent text-center mt-2 border border-b-4 border-yellow-600 hover:bg-yellow-600 hover:text-white active:border-yellow-400 rounded-lg py-2 px-4 w-40 sm:w-full sm:mx-2"
+                value={enteredValue.nextPurchase}
+              >
+                Kind of soon
+                <input
+                  className="radio"
+                  type="radio"
+                  name="nextPurchase"
+                  value="14"
+                  onChange={handleChange}
+                />
+              </label>
+              <label
+                className="bg-transparent text-center mt-2 border border-b-4  border-red-600 hover:bg-red-600 hover:text-white active:border-red-400 rounded-lg py-2 px-4 w-40 sm:w-full sm:mx-2"
+                value={enteredValue.nextPurchase}
+              >
+                Not so soon
+                <input
+                  className="radio"
+                  type="radio"
+                  name="nextPurchase"
+                  value="30"
+                  onChange={handleChange}
+                />
+              </label>
+            </div>
+            <div className="pointer-events-none absolute inset-y-0 right-0 flex items-center px-2 text-gray-700">
+              <svg
+                className="fill-current h-4 w-4"
+                xmlns="http://www.w3.org/2000/svg"
+                viewBox="0 0 20 20"
+              >
+                <path d="M9.293 12.95l.707.707L15.657 8l-1.414-1.414L10 10.828 5.757 6.586 4.343 8z" />
+              </svg>
             </div>
           </div>
 
-          <div className="flex items-center justify-between">
+          {enteredValue.nextPurchase && (
```

Comment:
Are there cases where this property doesn't have a value?

Reply:
ah, I was supposed to update this. Thanks for pointing this out.